dev-support/create-release/release-util.sh

```diff
@@ -745,6 +747,8 @@ function maven_set_version { #input: <version_to_set>
     log "${MVN[@]}" versions:set-property -Dproperty=revision -DnewVersion="$this_version" -DgenerateBackupPoms=false
     "${MVN[@]}" versions:set-property -Dproperty=revision -DnewVersion="$this_version" -DgenerateBackupPoms=false | grep -v "no value" # silence logs
   fi
+  log "${MVN[@]}" spotless:apply -DspotlessFiles='**/pom.xml'
```

Comment:
How could setting version in pom break the style?

Reply:
In theory it shouldn't. But then neither would generating release notes. Since we manage the structure of these files via spotless, i figure better safe than sorry.